lizmap/modules/presentation/classes/presentation.listener.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+class presentationListener extends jEventListener
+{
+    public function ongetMapAdditions($event)
+    {
+        $basePath = jApp::config()->urlengine['basePath'];
+
+        // Add JS and CSS for module
+        $jsCode = array();
+        $css = array();
+
+        // Check config
+        jClasses::inc('presentation~presentationConfig');
```

Comment:
store you presentationConfig class into a lib directory, with a namespace, and declare the namespace into the module.xml sot the class will be autoloadable, and you won't need to do `jClasses::inc()`. jClasses is deprecated.